blocks/init/src/blocks/components/hamburger/hamburger-style.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable max-nesting-depth */
+
+$hamburger: (
+  colors: (
+    line: #2C2C2C,
+  ),
+  width: 24px,
+  line-height: 2px,
+  padding: 8px
+);
+
+body {
```

Comment:
why wrapping this inside a body?

Reply:
Because I'm toggling the the hamburger through a class on body (as agreed). I don't need to wrap the first part in body but then I'll have 2 top level selectors in a single file (which is also an issue). 🤷‍♂

Reply:
That is ok. in this case. because it is full specific.

If you wrap the mail class then any overrides on that class will be hard because it has strength level 2 and with bem structure that is not ok. For the active state on body it is ok